### DISCUSSION TITLE
Mastodon Share API a little issue

### DISCUSSION THREAD

**Discussion by baobabresearch** [mastodon/mastodon#25380] [General]
I found a widget on the Internet so that visitors to my website can share news through Mastodon. I visited Shareaholic(.com) widget, entered my Mastodon instance. When I authorized my account through the Mastodon API and clicked the "Publish" button, there was no message "Post published", and the text I entered before simply disappeared from the text field. Users may not understand whether they posted an entry to their Mastoon page or not

Maybe Mastodon developers haven't added this notification yet, please add it

**Comment by baobabresearch** [comment]
I do not consider this a bug, but rather an aesthetic flaw, so I did not know which section to post in, I posted it here

**Comment by andypiper** [comment]
I'm not familiar with Shareaholic, but this seems like it might be something for that app to add, rather than an issue with the Mastodon API?

**Reply by andypiper** [reply]
No, this is my mistake. I see what you are describing, now. This is a request for the `/share` dialog to show when a status has been posted (rather than refreshing to blank). This has already been raised issue #11751 so I am going to suggest that you follow that issue.

**Reply by baobabresearch** [reply]
Ok, I'm sorry for duplicating.
